schema/deploy/shipping/views.sql

```diff
@@ -2911,4 +2913,198 @@ comment on view shipping.uw_reopening_ehs_reporting_v1 is
+        and testing_trigger is null
+        -- Filter for postive daily attestations only
+        and screen_positive
+    ),
+
+    -- Select enrollments for testing baseline and surveillance purposes
+    baseline_and_surveillance as (
+        select
+            uw_enrollments.redcap_project_id,
+            uw_enrollments.redcap_record_id,
+            uw_enrollments.redcap_event_name,
+            uw_enrollments.redcap_repeat_instance,
+            uw_enrollments.encountered,
+            individual,
+            tier,
+            latest_invite_date,
+            latest_collection_date,
+            case
+                when coalesce(latest_invite_date, latest_collection_date) is null and tier = '1' then 4
+                when coalesce(latest_invite_date, latest_collection_date) is null and tier in ('2', '3') then 5
+                when tier = '1' then 7
+                when tier in ('2', '3') then 8
+                else null
+              end as priority,
+            case
+                when coalesce(latest_invite_date, latest_collection_date) is null and tier = '1' then 'tier_1_baseline'
+                when coalesce(latest_invite_date, latest_collection_date) is null and tier in ('2', '3') then 'tier_2_and_3_baseline'
+                when tier = '1' then 'tier_1_surveillance'
+                when tier in ('2', '3') then 'tier_2_and_3_surveillance'
+                else null
+            end as priority_reason
+        from uw_enrollments
+        join uw_individuals using (individual)
+        -- Filter to enrollments that have never been invited to test or last invite was over 7 days before today
+        where (latest_invite_date is null or latest_invite_date < current_date - interval '7 days')
+        -- Filter to enrollments have never had a sample collected or last sample collection was over 7 days before today
+        and (latest_collection_date is null or latest_collection_date < current_date - interval '7 days')
+    ),
+
+    /**
+    Select encounters for surge testing purposes.
+    Handled separately from daily attestation encounters because we want to
+    include all surge selected participants regardless of previous testing.
+    **/
+    surge_testing as (
+        select
+            uw_encounters.redcap_project_id,
+            uw_encounters.redcap_record_id,
+            uw_encounters.redcap_event_name,
+            uw_encounters.redcap_repeat_instance,
+            uw_encounters.encountered,
+            individual,
+            tier,
+            latest_invite_date,
+            latest_collection_date,
+            6 as priority,
+            surge_selected as priority_reason
+        from uw_encounters
+        join uw_individuals using (individual)
+        join uw_enrollments using (individual)
+        where surge_selected is not null
+        and testing_trigger is null
+    )
+
+    select * from positive_daily_attestations
```

Comment:
We may want to wrap this with a big select *, min(priority) from <the union all queries> group by individual (and other columns) .... so that we return a given person only once.

Reply:
The inner view intentionally doesn't collapse by person; that's the job of the outer view below, which does:

```
select distinct on (individual)
```

Reply:
Right.